compose/extends.md

```diff
@@ -99,7 +99,7 @@ cache:
     - 6379:6379
 ```
 
-When you run `docker-compose up` it reads the overrides automatically.
+When you run `docker compose up` it reads the overrides automatically.
```

Comment:
Also missing for this file:
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L122
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L125
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L168

Reply:
It didn't look there was a change needed on line 122. If so let me know. L125, L168 replaced.

Reply:
My bad for the line 122 🤦‍♂️
I think we should change this one

- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L164

Reply:
Replaced.